### ISSUE TITLE
2.2.1 - No Find Button, Error Sensor or Battery Status

### ISSUE THREAD

**Issue by jaquesp** [apexad/homebridge-eufy-robovac#45]
Hi - installed 2.2.1 and configured for my Eufy C30.
Config UI X doesn't provide me with the options for Find/Error and despite adding them to the config, no such options show up in HomeKit. I also can't see any battery status - is this an issue?

{
    "name": "Vacuum Cleaner",
    "deviceId": "06868586807d3a1af4f2",
    "localKey": "b37a89883999ba27",
    "useSwitchService": false,
    "debugLog": false,
    **"hideFindButton": false,
    "hideErrorSensor": false,**
    "accessory": "Eufy RoboVac",
    "_bridge": {
        "username": "xxxx",
        "port": 35270
    }
}

**Comment by chrisswanda** [comment]
**EDIT** - never mind... it was removed.  https://github.com/apexad/homebridge-eufy-robovac/commit/a5d4ea1c3b36e5444042716188721e72ea1c8b69

Same here.  Not sure when it happened, but I used to have a Find Button defined.   Only a few weeks ago I was trying to find my vacuum and Siri told me I had no contact named Vacuum... checked out my bridge settings and there is no find button, even though I have it set to false in the config.
